src/modules/project/funding/state/pollingFundingTx.ts

```diff
@@ -0,0 +1,27 @@
+import { atom, useAtomValue, useSetAtom } from 'jotai'
+
+const POLLING_INTERVAL = 5 * 1000 // 5 seconds
```

Comment:
Would it be easy to set the first polling at 5 seconds, and then lower it to 1s. If the sub hasn't worked within 5 seconds, it likely won't work, so we can increase the poll frequency.

Reply:
The user would most likely take some time to open up wallet and get things ready to scan. So, I believe for a majority of the cases, the first 5 or 10 seconds since we start polling, won't have much activity in it. 

Maybe after 10 or 15 seconds, we can increase the frequency, what do you think about that?

Reply:
Good point, let's keep lower it after 15 but not to 1 then, something like 2 or 3 should be good enough.